flow-shop/chapter.md

```diff
@@ -248,8 +249,9 @@ To make locating the correct instance easier, we assume that every line will be
             sys.exit(0)
 ```
 
-Finally, we parse the data directly -- converting each of the tasks processing times to an integer and storing them in a list. As a final step, we zip the data to invert the rows and columns so that the format respects what is expected by the solving code above (i.e., every item in data should correspond to a particular job).
+Finally, we parse the data directly, converting each of the tasks' processing times to an integer and storing them in a list. Finally, we zip the data to invert the rows and columns so that the format respects what is expected by the solving code above. (That is, every item in `data` should correspond to a particular job.)
```

Comment:
Well, s' is the standard way to signify the possessive case for plural nouns in English:  parents' house, dogs' tails. You should probably try to get used to it. :) 

However, it might be bothering you particularly in this case because there is some question as to whether the noun with "each" is [singular or plural](http://www.grammar.com/each-singular-or-plural/). I made a mistake here; we we should have:

> converting each task's processing time to an integer and storing it in a list.

Would that be okay with you?

Thanks for catching this!